adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -34,24 +34,43 @@
 from adafruit_esp32spi import adafruit_esp32spi
 import adafruit_esp32spi.adafruit_esp32spi_requests as requests
 
+class WiFiConnType: # pylint: disable=too-few-public-methods
```

Comment:
Instead of using a separate class, you could just add these constants to `ESPSPI_WiFiManager`.

Reply:
Agreed. After I put the class together, I thought to myself that this seemed like overkill for 2 options.